lisa/sut_orchestrator/azure/tools.py

```diff
@@ -192,8 +197,10 @@ def can_install(self) -> bool:
             and self.node.os.information.version < "7.8.0"
         ):
             return True
-
-        return False
+        raise SkippedException(
```

Comment:
Don't skip test case in the tool, catch it and use Skipped in test case.

Reply:
updated